hetmech/diffusion.py

```diff
@@ -90,7 +90,7 @@ def diffusion(
         adjacency_matrix = diffusion_step(
             adjacency_matrix, row_damping, column_damping)
 
-        node_scores = adjacency_matrix @ node_scores
+        node_scores = node_scores @ adjacency_matrix
```

Comment:
Currently `node_scores` is initialized as a row vector, which works for our current purposes.
No need to make a change here.